backend/src/nodes/nodes/utility/convert_number.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import NumberInput, TextInput
+from ...properties.outputs import NumberOutput
+from . import category as UtilityCategory
+
+
+@NodeFactory.register("chainner:utility:convert_number")
+class TextLengthNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Converts a text to base-10."
+        self.inputs = [
+            TextInput("Text", min_length=0),
+            NumberInput("Base", default=10, minimum=0, maximum=64),
```

Comment:
```py
>>> int("3", 37)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: int() base must be >= 2 and <= 36, or 0
```

As in most languages, integer parsing only goes up to base 36. This is because these parsers are case-insensitive. E.g. in base 16, both `F` and `f` are parsed as 15. 

Since base 1 doesn't exist and "base 0" is highly specific to python, I would also disallow those.

```suggestion
            NumberInput("Base", default=10, minimum=2, maximum=36),
```